### PR TITLE
feat: adds dependabot configuration

### PR DESCRIPTION
**Pull request by baywet**
Signed-off-by: Vincent Biret <vibiret@microsoft.com>

### CODE REVIEW

**Comment by baywet** [comment]
depends on #12 to be merged in first, and a rebase later, especially if we want to avoid conflicts

File: .github/workflows/npm-publish.yml
Comment (gavinbarron) [inline .github/workflows/npm-publish.yml:1]: __nit__: let's rename this file to npm-ci or something similar, it's not publishing a package

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Node.js Package
```
Comment (gavinbarron) [inline .github/workflows/npm-publish.yml:1]: __nit:__ CI Build maybe a better name?

Comment (baywet) [reply .github/workflows/npm-publish.yml:1]:
```suggestion
name: CI build
```

**Review by thewahome — APPROVED** [review]

**Pull request merged**